tests/jvm/src/test/resources/metac-metacp.diff

```diff
@@ -291,13 +291,13 @@ com.javacp.ClassSuffix#Inner#`<init>`().
 -  tag: PUBLIC
 -  symbol: ""
 +  tag: PRIVATE_WITHIN
-+  symbol: "com.javacp."
++  symbol: "com/javacp/"
```

Comment:
Why did this change?

Reply:
The change is expected, it used to be `com.javacp.` and is now `com/javacp/`. Observe that the diff is only one line, the `PRIVATE_WITHIN` diff is from the diff file itself, it's unchanged in this PR.

Reply:
Ah right.